### DISCUSSION TITLE
DIANN exiting before processing files

### DISCUSSION THREAD

**Discussion by taganief** [vdemichev/DiaNN#547] [General]
Hello

I started using a new desktop for DIA-NN;
I successfully generated a new library with this desktop
When I try to process some files, DIA-NN exists as seen in the log below.

I tried files and a library previously used (successfully) on a different computer and the result is the same. Does DIA-NN need some package that I dont have on the new computer even though I used it for library generation? 
Although it says a report is generate, there is none.

[0:16] Library contains 249695 proteins, and 165932 genes
[0:19] Spectral library loaded: 251247 protein isoforms, 412892 protein groups and 14071271 precursors in 8210271 elution groups.
[0:19] Initialising library

[0:25] First pass: generating a spectral library from DIA data
[0:25] File #1/2
[0:25] Loading run C:\data_proc\nina_100\20220428_DEPO_sample1.wiff

DIA-NN exited
DIA-NN-plotter.exe "C:\DIA-NN\1.8\report.stats.tsv" "C:\DIA-NN\1.8\report.tsv" "C:\DIA-NN\1.8\report.pdf"
PDF report will be generated in the background

Thanks,
Tariq

**Comment by vdemichev** [comment]
Hi Tariq,

Sciex libs were not copied into the DIA-NN folder, please see here https://github.com/vdemichev/DiaNN#raw-data-formats
Also, we recommend using DIA-NN 1.8.1

Best,
Vadim

**Comment by taganief** [comment]
Thank you!
I have 1.8.1 but the same result of course - same with the beta you sent me
Thanks for the quick response - I forgot this requirement!

Tariq

On Tue, 1 Nov 2022 at 17:19, Vadim Demichev ***@***.***>
wrote:

> Hi Tariq,
>
> Sciex libs were not copied into the DIA-NN folder, please see here
> https://github.com/vdemichev/DiaNN#raw-data-formats
> Also, we recommend using DIA-NN 1.8.1
>
> Best,
> Vadim
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vdemichev/DiaNN/discussions/547#discussioncomment-4028461>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHKERB2KWHEMMMWZD4DD4QTWGEYH7ANCNFSM6AAAAAARUE6PQA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Best,
Tariq

**Comment by taganief** [comment]
its running now; Thank you!

Did you perhaps see a previous mail  I sent you regarding RAM usage for a
few vs many files?

Tariq

On Tue, 1 Nov 2022 at 17:22, Tariq Ganief ***@***.***> wrote:

> Thank you!
> I have 1.8.1 but the same result of course - same with the beta you sent me
> Thanks for the quick response - I forgot this requirement!
>
> Tariq
>
> On Tue, 1 Nov 2022 at 17:19, Vadim Demichev ***@***.***>
> wrote:
>
>> Hi Tariq,
>>
>> Sciex libs were not copied into the DIA-NN folder, please see here
>> https://github.com/vdemichev/DiaNN#raw-data-formats
>> Also, we recommend using DIA-NN 1.8.1
>>
>> Best,
>> Vadim
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/vdemichev/DiaNN/discussions/547#discussioncomment-4028461>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AHKERB2KWHEMMMWZD4DD4QTWGEYH7ANCNFSM6AAAAAARUE6PQA>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
> Best,
> Tariq
>


-- 
Best,
Tariq

**Comment by johnpleid** [comment]
Hi I am experiencing the same problem with the beta version 23with silac data 
how can i solve?
thank you
